functional/js/fibonacci.js

```diff
@@ -5,7 +5,20 @@
  */
 function generateFibonacciUsingLoop(sequence) {
   // write your code here
-  return [];
+  let fibonacci = [0, 1];
+
+  if (sequence == 1) {
+    fibonacci.pop();
+    return fibonacci;
+  } else if (sequence == 2) {
+    return fibonacci;
+  }
```

Comment:
If you have ```else if``` code with ```return``` statement like this, you can make it more readable by writing it this way.
```js
let fibonacci = [0, 1];

if (sequence == 2) {
  return fibonacci;
}

if (sequence == 1) {
  fibonacci.pop();

  return fibonacci;
}
```